workers/ohsome_quality_analyst/indicators/mapping_saturation/indicator.py

```diff
@@ -35,162 +45,183 @@ def __init__(
             layer_name=layer_name,
             feature=feature,
         )
-        self.time_range = time_range
+        self.time_range: str = time_range
+
         # The following attributes will be set during the life-cycle of the object.
         # Attributes needed for calculation
         self.values: list = []
-        self.values_normalized: list = []
+        self.latest_value: Union[None, int, float] = None
         self.timestamps: list = []
-        self.no_data: bool = False
-        self.deleted_data: bool = False
+
+        self.upper_threshold = 0.97  # Threshold derived from Gröchenig et al.
+        # TODO: What is a good lower threshold?
```

Comment:
Is this already an issue somewhere?

Reply:
Just created one 
https://github.com/GIScience/ohsome-quality-analyst/issues/248